### PR TITLE
Adding imgages [WAIV-1123]

### PR DESCRIPTION
**Pull request by konstantin1390**
Fixes #

### Changes

Summary of changes you made.

* Change 1
* Change 2

### Test plan

Explain how to test changes you made.

* [ ] Step 1
* [ ] Step 2

### Demo (optional)

Sreenshots (before & after) or video that show result of your work.

### CODE REVIEW

File: src/client/components/Editor/EditorInput.js [hunk 1]
```diff
@@ -318,13 +363,17 @@ class EditorInput extends React.Component {
...
-          insertImage={this.beforeInsertImage}
-          onImageUpload={onImageUpload}
-          onImageInvalid={onImageInvalid}
-        />
+          okButtonProps={{ disabled: isLoadingImage }}
+          cancelButtonProps={{ disabled: isLoadingImage }}
+          visible={showModal}
+          onOk={this.handleOnOkModal}
+        >
+          <ImageSetter getImages={this.getImages} onLoadingImage={this.onLoadingImage} />
```
Comment (milklim) [inline src/client/components/Editor/EditorInput.js]: rename prop getImages => onImageLoaded

File: src/client/components/ImageSetter/ImageSetter.js [hunk 2]
```diff
@@ -25,11 +56,59 @@ const ImageSetter = ({
         name: filename,
         id: uuidv4(),
       };
-      handleAddImageByLink(newImage);
+      const img = new Image();
```
Comment (milklim) [inline src/client/components/ImageSetter/ImageSetter.js]: Remove helper with this functionality

File: src/client/components/ImageSetter/ImageSetterHelpers.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { message } from 'antd';
```
Comment (milklim) [inline src/client/components/ImageSetter/ImageSetterHelpers.js]: remove this

File: src/client/object/AppendForm.js [hunk 4]
```diff
@@ -546,13 +547,17 @@ export default class AppendForm extends Component {
...
-  handleAddImageByLink = image => {
+  getImageByLink = image => {
     const { getFieldValue } = this.props.form;
     const currentField = getFieldValue('currentField');
-    this.setState({ imageUploading: false, currentImage: [image] });
+    this.setState({ currentImage: [image], imageUploading: false });
     this.props.form.setFieldsValue({ [currentField]: image.src });
   };
 
+  handleAddImageByLink = image => {
```
Comment (milklim) [inline src/client/object/AppendForm.js]: check if you need this function

File: src/client/components/Editor/EditorInput.js (hunk 1 above)
Comment (konstantin1390) [reply src/client/components/Editor/EditorInput.js]: ++

File: src/client/components/ImageSetter/ImageSetterHelpers.js (hunk 3 above)
Comment (konstantin1390) [reply src/client/components/ImageSetter/ImageSetterHelpers.js]: ++

File: src/client/components/ImageSetter/ImageSetter.js (hunk 2 above)
Comment (konstantin1390) [reply src/client/components/ImageSetter/ImageSetter.js]: ++

File: src/client/object/AppendForm.js (hunk 4 above)
Comment (konstantin1390) [reply src/client/object/AppendForm.js]: ++

**Pull request merged**